roles/online_docs/templates/mkdocs/docs/logins-windows.md

```diff
@@ -39,55 +47,121 @@ If you prefer another terminal application consult the corresponding manual.
 
  * SSH jump hosts popup window
     * 5: _Gateway host_ field: Use _**{{ first_jumphost_address }}**_ for the _Jumphost_ address.
-    * Optional: _Port_ field: The default port for SSH is _**22**_ and this is usually fine. 
+    * Optional: _Port_ field: The default port for SSH is _**22**_ and this is usually fine.
       However if you encounter a network where port 22 is blocked, you can try port 443. (Normally used for HTTPS, but our Jumposts can use it for SSH too.)
     * 6: _Username_ field: Use your _**account name**_ as you received it by email from the helpdesk (same as for 3).
     * 7: Select _Use SSH key_ and
-    * 8: Click the small button to select the _**private key file**_ you generated previously (same as for 4).  
+    * 8: Click the small button to select the _**private key file**_ you generated previously (same as for 4).
       **Important**: the path to the selected private key will be shown.
-      Depending on how you browsed to the private key file, the path may 
-        * Either start with a drive letter, colon and single backslash.  
-          E.g. ```H:\path\to\private_key.ppk```  
+      Depending on how you browsed to the private key file, the path may
+        * Either start with a drive letter, colon and single backslash.
+          E.g. ```H:\path\to\private_key.ppk```
           This is fine and should work.
-        * Or start with two backslashes.  
-          E.g. ```\\path\to\private_key.ppk```  
-          This won't work and MobaXterm will fail silently: no login, no error, no nothing.  
+        * Or start with two backslashes.
+          E.g. ```\\path\to\private_key.ppk```
+          This won't work and MobaXterm will fail silently: no login, no error, no nothing.
```

Comment:
The trailing spaces are optional, but the trailing double spaces were put there explicitly the create line breaks and improve the readability.